### PR TITLE
Added NDLA.no

### PR DESCRIPTION
**Pull request by Foorack**
NDLA is the main digital learning platform for higher education schools in Norway.

### CODE REVIEW

**Comment by jeremyn** [comment]
Please add more subdomains, see [here](https://www.google.com/search?q=site%3A*.ndla.no&ie=utf-8&oe=utf-8#q=site:*.ndla.no+-site:www.ndla.no+-site:ndlap4.ndla.no+-site:bak.ndla.no+-site:arkiv.ndla.no+-site:*.arkiv.ndla.no+-site:om.ndla.no).

File: src/chrome/content/rules/ndla.no.xml [hunk 1]
```diff
@@ -12,6 +12,10 @@
 	<target host="ndla.no" />
 	<target host="www.ndla.no" />
 	<target host="ndlap4.ndla.no" />
+	<target host="ndlap4.ndla.no" />
+	<target host="ndlap4.ndla.no" />
+	<target host="ndlap4.ndla.no" />
+	<target host="ndlap4.ndla.no" />
```
Comment (Foorack) [inline src/chrome/content/rules/ndla.no.xml]: More subdomains added :D 
Jk, will fix in a sec.

**Comment by Foorack** [comment]
@jeremyn Updated.

**Comment by jeremyn** [comment]
Thanks. Please:

Mark as `Refused`:
- [ ] https://bak.ndla.no

Mark as `Mismatch`:
- [ ] https://arkiv.ndla.no
- [ ] https://2009.arkiv.ndla.no
- [ ] https://2010.arkiv.ndla.no
- [ ] https://2011.arkiv.ndla.no
- [ ] https://2012.arkiv.ndla.no
- [ ] https://i.ndla.no
- [ ] https://om.ndla.no
- [ ] https://static.ndla.no

Mark as `Different content`:
- [ ] https://samskrive.ndla.no
- [ ] https://sti.ndla.no

Mark as `Timeout`:
- [ ] https://nettskole.ndla.no
- [ ] https://tokt.ndla.no

Add target:
- [ ] https://alerter.ndla.no
- [ ] https://auth.ndla.no
- [ ] https://cdntest-a.ndla.no
- [ ] https://cdntest-b.ndla.no
- [ ] https://cdntest-c.ndla.no
- [ ] https://feedback.ndla.no
- [ ] https://min.ndla.no
- [ ] https://ndlap4cpanelhost01v.ndlap4.ndla.no

**Comment by Foorack** [comment]
Updated pull https://github.com/EFForg/https-everywhere/pull/6882.

**Comment by jeremyn** [comment]
I have two main approaches to finding domains.

1. Google search trick. Note that sometimes Google doesn't show all domains even when you think it should. For example if I remember correctly, `static.` randomly appeared at the end of some elimination query when I didn't expect it.

2. When I try sites, I usually have the Firefox developer tools `Network` tab open and watch what is loaded. If any new sites jump out at me, I might note them down. For example loading https://ndla.no you can see calls to https://auth.ndla.no , https://cdntest-a.ndla.no and others.

Also if I happen across some new domain while reviewing, I might note it. For example, when I go to https://arkiv.ndla.no , I get the below mismatch message which reveals https://ndlap4cpanelhost01v.ndlap4.ndla.no :

```
arkiv.ndla.no uses an invalid security certificate.

The certificate is only valid for the following names:
  ndlap4cpanelhost01v.ndlap4.ndla.no, www.ndlap4cpanelhost01v.ndlap4.ndla.no

Error code: SSL_ERROR_BAD_CERT_DOMAIN
```

Another thing you can do is look for some sort of sitemap, like [this one for eBay](http://pages.ebay.com/sitemap.html). I don't use these when I'm reviewing though.

I think it's important to be realistic and not go overboard looking. I don't want you to feel like making rulesets is a huge burden or weird puzzle. I'm not aggressively trying to find domains that you missed. I do the Google search trick, watch sites Ioad, and sometimes make notes of new domains that I discover. If the domain looks really "internal" to the site then I might skip it. For example if I go to https://ndlat4vhosts02v.ndlap4.ndla.no , I get a mismatch message that includes:

```
The certificate is only valid for the following names:
  auth.test.ndla.no, auth.test2.ndla.no, www.auth.test2.ndla.no, www.auth.test.ndla.no
```

I don't think it's necessary to look into domains like `www.auth.test2.ndla.no`. Another example, I don't know how you found https://ndlap4revproxy01v.ndlap4.ndla.no but if you used some kind of script or hunted through DNS listings then that's too much.

Thanks for your work. It does help us the more thorough you are but I don't want you to feel so self-conscious or nervous about it that you stop contributing or feel bad about contributing.

**Pull request closed without merging**